lib/daylight.cpp

```diff
@@ -840,6 +859,9 @@ bool Daylight::simStandard(int blindGroupNum, int setting, Control *model){
         sunsOct=model->spaceDirectory()+model->intermediateDataDirectory()+model->spaceName()+"_"+model->windowGroups()[blindGroupNum].name()+"_sun_set"+std::to_string(setting+1)+"_std.oct";
 
     }
+    if (writeCL){
+            std::cout<<"***\nCreating suns octree.\n***"<<std::endl;
```

Comment:
@jasondegraw I think it would be a good idea to have a debugging message. These were added to debug, so that makes sense to me.  I have been thinking about that for a long time now.

Thanks,
Craig

> On Mar 30, 2016, at 4:39 PM, Jason DeGraw <notifications@github.com> wrote:
> 
> In lib/daylight.cpp:
> 
> > @@ -840,6 +859,9 @@ bool Daylight::simStandard(int blindGroupNum, int setting, Control *model){
> >          sunsOct=model->spaceDirectory()+model->intermediateDataDirectory()+model->spaceName()+"_"+model->windowGroups()[blindGroupNum].name()+"_sun_set"+std::to_string(setting+1)+"_std.oct";
> >  
> >      }
> > +    if (writeCL){
> > +            std::cout<<"***\nCreating suns octree.\n***"<<std::endl;
> I'm not sure that these messages are a great idea. Can we integrate them into the logging setup that we have? If we need to add a "debug" message type, we can do that.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>